templates/common/azure/units/ovs-configuration.service.yaml

```diff
@@ -0,0 +1,21 @@
+name: ovs-configuration.service
+enabled: {{if eq .NetworkType "OVNKubernetes" "OpenShiftSDN"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Configures OVS with proper host networking configuration
+  # This service is used to move a physical NIC into OVS and reconfigure OVS to use the host IP
+  Requires=openvswitch.service
+  Wants=NetworkManager-wait-online.service
+  After=firstboot-osupdate.target
+  After=NetworkManager-wait-online.service openvswitch.service network.service nodeip-configuration.service
+  Before=kubelet-dependencies.target node-valid-hostname.service dnsmasq.service
```

Comment:
I know this is not new at all in this codebase, but the rampant copying of systemd units is a problem.  systemd supports proper "drop-ins" so I believe we should make it work to do e.g.:

`templates/common/azure/units/ovs-configuration.service.d/20-azure-dnsmasq.conf` which just has:
```
[Unit]
Before=dnsmasq.service
```
and that's it...

Reply:
Just to confirm if I wanted to go back in and improve this, that has scaffolding to create that path or do I need to do something to create the `ovs-configuration.service.d` dropin directory?